src/Razor/test/Microsoft.NET.Sdk.Razor.Test/IntegrationTests/BuildIncrementalismTest.cs

```diff
@@ -28,7 +28,7 @@ public async Task BuildIncremental_SimpleMvc_PersistsTargetInputFile()
             var thumbprintLookup = new Dictionary<string, FileThumbPrint>();
 
             // Act 1
-            var result = await DotnetMSBuild("Build", runRestoreBeforeBuildOrPublish: false);
+            var result = await DotnetMSBuild("Build");
```

Comment:
Do none of the tests that require a rename need to be restored? Does that just work?

Reply:
I changed the rename tests to manually restore (they do `DotnetMSBuild("Restore")` followed by a `DotnetMSBuild("Build")`) a while back. As long as the MSBuild bits continue to work when copied to a temp directory all should be fine.